PerimeterXModule/Internals/Validators/PXCookieValidator.cs

```diff
@@ -1,33 +1,69 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Text.RegularExpressions;
 
 namespace PerimeterX
 {
-	class PXCookieValidator : IPXCookieValidator
+	public class PXCookieValidator : IPXCookieValidator
 	{
-		private PxModuleConfigurationSection config;
+		public static readonly string CALL_REASON_INVALID_COOKIE = "invalid_cookie";
+		public static readonly string CALL_REASON_DECRYPTION_FAILED = "cookie_decryption_failed";
+		public static readonly string CALL_REASON_VALIDATION_FAILED = "cookie_validation_failed";
+		public static string CALL_REASON_NO_COOKIE = "no_cookie";
+		public static string CALL_REASON_EXPIRED_COOKIE = "cookie_expired";
+		public static string CALL_REASON_SENSITIVE_ROUTE = "sensitive_route";
+		public static string CALL_REASON_MOBILE_ERROR = "mobile_error_{0}";
```

Comment:
Note that the design doc states `mobile_sdk_<number>`. Assuming that here is the correct convention, please update the design doc to reflect it as well (make sure future enforcers use the right s2d call reason convention).

Reply:
👍